### ISSUE TITLE
minimum() returns wrong minat value

### ISSUE THREAD

**Issue by Bort-Plate** [MajenkoLibraries/Average#9]
Observe the code comments below to see the issue:

```
Average<int> ave(5);
void loop()
{
int minat = 0;
	ave.push(-4); // If this value is the minimum, minat=0
	ave.push(5); // If this value is the minimum, minat=4
	ave.push(5); // If this value is the minimum, minat=3
	ave.push(5); // If this value is the minimum, minat=2
	ave.push(5); // If this value is the minimum, minat=1
	Serial.print("min: "); Serial.println(ave.minimum(&minat));
	Serial.print("minat: "); Serial.println(minat);
}
```

The first value in the buffer is at 0, but then counts down from 4 to 1.

**Comment by majenkotech** [comment]
It looks like a problem with the `get()` function - if the buffer isn't full it can't seem to get a value for entry 0.

**Comment by majenkotech** [comment]
Fixed in commit 32a4f32
